koala/ast/graph.py

```diff
@@ -413,13 +453,14 @@ def __init__(self,value,precedence,associativity):
         self.precedence = precedence
         self.associativity = associativity
 
-def shunting_yard(expression, named_ranges):
+def shunting_yard(expression, named_ranges, ref = ''):
```

Comment:
This is the only way I found to pass the cell address (from which formula is evaluated) down to the actual compiled python code.
This is needed because Excel range basic operations (+, -, * ...) are applied on matching cells.

Example:
on cell **C2**, i write the formula **A1:A3 + B1:B3**.
This will actually compute **A2 + B2**, because the formula is relative to cell **C2**.

This cell reference needs to be passed from actual cell reading to Range functions signatures, and so, written while passing through `OperatorNode` https://github.com/iOiurson/koala/blob/master/koala/ast/graph.py#L292.